charts/vald-benchmark-operator/templates/_helpers.tpl

```diff
@@ -0,0 +1,310 @@
+{{/* vim: set filetype=mustache: */}}
```

Comment:
@kpango 
It is not a specialized helper.

> Since there are many parts in common with Origin's helper, I'd like to copy and adapt or automate the Originization and automation of the source.

IMO, It would be better to use helm library charts or automate updating workflow after discussing.
However there will be many changes, so I'd like to use a copy of `charts/templates/_helper.tpl`  for now.